### PR TITLE
add OS UnitTests github workflow

### PR DESCRIPTION
**Pull request by jakebolewski**

### CODE REVIEW

**Comment by jakebolewski** [comment]
bors try

**Comment by skandalaCLIMA** [comment]
bors try

File: bors.toml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 status = [
   "CliMA.RRTMGP.jl"
```
Comment (charleskawczynski) [inline bors.toml:2]:
```suggestion
  "CliMA.RRTMGP.jl",
```

**Comment by skandalaCLIMA** [comment]
bors try

File: bors.toml (hunk 1 above)
Comment (jakebolewski) [reply bors.toml]: :+1:

**Comment by skandalaCLIMA** [comment]
bors try

**Comment by skandalaCLIMA** [comment]
bors try

**Comment by skandalaCLIMA** [comment]
bors try-

**Comment by skandalaCLIMA** [comment]
bors try

**Comment by jakebolewski** [comment]
bors try

**Comment by jakebolewski** [comment]
bors try-

**Pull request merged**